### ISSUE TITLE
Bound events drop after 7 pages

### ISSUE THREAD

**Issue by TimOgilvy** [blasten/turn.js#418]
Events bound to page elements using jquery's event binding are dropped once the pages are out of range by 7+.

Given that turn.js is marketed as a jquery plugin, this lack of compatibility with basic jquery functionality doesn't seem congruent. Can we get a fix, or an answer as to why there isn't one please.

**Comment by alQemist** [comment]
Why would you need to bind to pages that are not within the current pages of pages which are maintained effectively as cached pages ? Or do I misunderstand your question ? Pages out of range I believe are removed automatically from cache….

**Comment by TimOgilvy** [comment]
Correct. The pages are removed, and all bindings are dropped, meaning that you have to rebind everything on the turned or turning events which is just inefficient in my view.

I believe there  is a way to bind events that won't drop, but it's not documented.

My point is that turn.js should have a handler for this, so that developers are able to implement it easily. Failing that it should at least be clearly documented with clear solutions. It's not.

**Comment by alQemist** [comment]
What version of Jquery are you using ?  Seems to me that using the latest version of JQuery gives you a nice method using  $(document).on('click','#myElement', myFunction);

**Comment by TimOgilvy** [comment]
If you're referring to jQuery 2.x it drops legacy support I can't afford to lose. I'm pretty sure I'm using 1.9.

**Comment by TimOgilvy** [comment]
I was using 'on' but not specifying a selector like you did.  It does work now. 

http://stackoverflow.com/questions/19032597/javascript-event-binding-persistence

I still think this should be more explicit in  the docs...

**Comment by quarrelsome** [comment]
Thanks people for handing me the solution on a platter. For the record I just burnt like four hours trying to work out what the hell was going wrong. Obviously no errors or any sort of clues and seemingly unusual behaviour.
It took me adding an "onclick" attribute to my elements to realise it was just caching the HTML text as opposed to any jQuery state as well (at first I burnt a while thinking something was e.PreventDefault() or otherwise intercepting the click).

If you want idiots like me to able to use the library without such burn time then I highly recommend making this clearer somehow. I genuinely thought I was losing my mind and had completely forgotten about how the cache worked in turn.js.

In terms of ideas: perhaps add an option for the page length of the cache? This option may not be specifically useful but it wedges the subject of the cache into a prominent position within the documentation. Also if people are just bashing buttons to see what fixes their issue it gives them a handle on the problem.
